src/pages/billing/List.js

```diff
@@ -401,15 +399,19 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
         loadAccounts,
         onDismiss: () => setCreatingBillingProject(null),
         onSuccess: billingProjectName => {
-          Ajax().Metrics.captureEvent(Events.billingCreationGCPBillingProjectCreated, { billingProject: billingProjectName })
+          Ajax().Metrics.captureEvent(Events.billingCreationGCPBillingProjectCreated, {
+            billingProjectName, billingProjectCloudPlatform: cloudProviders.gcp.label
```

Comment:
Changed to `billingProjectName` for consistency.

![image](https://user-images.githubusercontent.com/484484/209217184-135e4ff4-20c2-4f25-907f-2d25dc23fa8b.png)